demo/date-time-picker-localization-demos.html

```diff
@@ -0,0 +1,112 @@
+<dom-module id="date-time-picker-localization-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <h3>Localizing</h3>
+    <vaadin-demo-snippet id="date-time-picker-localizing" when-defined="vaadin-date-time-picker">
+      <template preserve-content>
+        <vaadin-date-time-picker label="Syntymäpäivä"></vaadin-date-time-picker>
```

Comment:
nit: birthday is a bit odd label for this when you're also picking the time

Reply:
I couldn't think of a better use case. I was thinking "birth date and time" to specify exact time of birth (like in the "Initial position of date picker" demo).

Would it be ok if I just update the label to "Syntymäpäivä ja -aika"?

Reply:
That would be ok 👍

Reply:
Updated.